dessert/rewrite.py

```diff
@@ -211,6 +211,13 @@ def get_data(self, pathname):
         with open(pathname, 'rb') as f:
             return f.read()
 
+    def get_resource_reader(self, name: str) -> importlib.abc.TraversableResources:
+        from types import SimpleNamespace
+        if sys.version_info < (3, 11):
+            from importlib.readers import FileReader
+        else:
+            from importlib.resources.readers import FileReader
+        return FileReader(SimpleNamespace(path=self._rewritten_names[name]))
```

Comment:
Is there a reason you didn't use the latest code from pytest:
```python
    if sys.version_info >= (3, 10):
        if sys.version_info >= (3, 12):
            from importlib.resources.abc import TraversableResources
        else:
            from importlib.abc import TraversableResources

        def get_resource_reader(self, name: str) -> TraversableResources:
            if sys.version_info < (3, 11):
                from importlib.readers import FileReader
            else:
                from importlib.resources.readers import FileReader

            return FileReader(types.SimpleNamespace(path=self._rewritten_names[name]))
```
https://github.com/pytest-dev/pytest/blob/main/src/_pytest/assertion/rewrite.py